common/locales/en/groups.json

```diff
@@ -37,7 +37,10 @@
     "invitedToNewParty": "You were invited to join a party! Do you want to leave this party and join <%= partyName %>?",
     "joinNewParty": "Join New Party",
     "declineInvitation": "Decline Invitation",
-    "loadingNewParty": "Your party is loading. Please wait...",
+    "partyLoading1": "Your party is being summoned. Please wait...",
+    "partyLoading2": "Your party is coming in from battle. Please wait...",
+    "partyLoading3": "Your party is gathering. Please wait...",
+    "partyLoading4": "Your party is materializing. Please wait...",
```

Comment:
As soon as we get the OK from @Alys and @lemoness on the wording for these, it can be merged.